terraform/deployments/cloudfront/main.tf

```diff
@@ -343,3 +352,53 @@ resource "aws_cloudfront_distribution" "assets_distribution" {
     minimum_protocol_version = "TLSv1.2_2021"
   }
 }
+
+import {
+  to = aws_lambda_function.url_rewrite
```

Comment:
This is causing the plan to fail in staging, because it looks like the function doesn't already exist there? If that's right, and intentional, we should probably add a `for_each = var.govuk_environment == "production"? [1] : []` type conditional to this.


iirc you can't use count here, but using a single element array and an empt array are usable substitutes. We're going to remove the import statement in a subsequent commit anyway.